cmd/generator.go

```diff
@@ -27,6 +29,8 @@ var (
 	queryMaxAge            = kingpin.Flag("query-max-age", "How back in the past metrics can be queried at most.").Default("24h").Duration()
 	tenantsCount           = kingpin.Flag("tenants-count", "Number of tenants to fake.").Default("1").Int()
 	seriesCount            = kingpin.Flag("series-count", "Number of series to generate for each tenant.").Default("1000").Int()
+	oooSeriesCount         = kingpin.Flag("out-of-order-series-count", "Number of out of order series to generate for each tenant.").Default("0").Int()
+	maxOOOTime             = kingpin.Flag("max-out-of-order-time", "How old an out of order sample can be *in minutes*.").Default("40").Int()
```

Comment:
Default to 40m because with 30m of OOO Allowance, we will be able to test if limits are working properly or not